models/public/parking-vehicle-detection/parking-vehicle-detection.md

```diff
@@ -0,0 +1,81 @@
+# Cascade-RCNN
+
+## Use Case and High-Level Description
+  The "Cascade-RCNN" model is two-stage multibox Detection network intended to perform 
+  object detection. This model is implemented using mmdetection framework.
+
+  The model input is a blob that consists of a single image of 1x3x800x1333 in BGR
+  order. The BGR mean values need to be subtracted
+  as follows: [123.675, 116.28, 103.53] before passing the image blob into the network. 
+  Also BGR values need to be scaled by division on following coefficients: [58.395, 57.12, 57.375]  
+
+
+## Example
```

Comment:
we are removing empty sections from model documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please provide sample image or remove empty section